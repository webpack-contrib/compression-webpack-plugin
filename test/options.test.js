/* eslint-disable */
import Plugin from '../src';
import webpack from './helpers/compiler';
import { assets } from './helpers/stats';

describe('options', () => {
  test('asset', () => {
    const config = { plugins: [ new Plugin() ] };

    return webpack('./fixture.js', config)
      .then((stats) => assets(stats, 2))
      .then((asset) => {
        const { source, map } = asset

        expect(source).toMatchSnapshot();
        expect(map).toMatchSnapshot();
      })
      .catch((err) => err);
  });
});
