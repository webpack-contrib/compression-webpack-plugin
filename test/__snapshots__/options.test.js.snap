// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`options asset 1`] = `
Object {
  "data": Array [
    31,
    139,
    8,
    0,
    0,
    0,
    0,
    0,
    2,
    19,
    109,
    144,
    205,
    106,
    195,
    48,
    16,
    132,
    239,
    122,
    138,
    69,
    189,
    200,
    198,
    216,
    166,
    244,
    84,
    147,
    23,
    40,
    9,
    45,
    133,
    158,
    140,
    17,
    178,
    188,
    73,
    149,
    216,
    146,
    170,
    159,
    198,
    33,
    248,
    221,
    171,
    52,
    41,
    244,
    144,
    211,
    194,
    176,
    59,
    223,
    206,
    28,
    177,
    183,
    66,
    30,
    94,
    188,
    209,
    150,
    181,
    117,
    87,
    180,
    164,
    202,
    161,
    134,
    188,
    74,
    51,
    207,
    43,
    96,
    219,
    168,
    101,
    80,
    70,
    179,
    201,
    12,
    113,
    196,
    2,
    56,
    63,
    94,
    143,
    56,
    206,
    214,
    184,
    224,
    57,
    255,
    47,
    58,
    252,
    138,
    202,
    33,
    231,
    25,
    156,
    9,
    161,
    209,
    35,
    248,
    224,
    148,
    12,
    180,
    33,
    175,
    253,
    30,
    101,
    40,
    7,
    220,
    42,
    141,
    111,
    206,
    88,
    116,
    225,
    196,
    238,
    26,
    82,
    206,
    209,
    111,
    126,
    145,
    180,
    128,
    51,
    124,
    139,
    49,
    226,
    51,
    4,
    23,
    17,
    150,
    172,
    33,
    210,
    104,
    31,
    96,
    134,
    21,
    60,
    61,
    54,
    132,
    252,
    125,
    9,
    66,
    8,
    96,
    243,
    133,
    13,
    224,
    48,
    68,
    167,
    97,
    110,
    200,
    66,
    46,
    177,
    62,
    133,
    155,
    140,
    62,
    65,
    226,
    139,
    56,
    6,
    184,
    226,
    82,
    214,
    123,
    153,
    90,
    122,
    91,
    163,
    93,
    130,
    176,
    228,
    155,
    168,
    228,
    214,
    202,
    146,
    145,
    84,
    85,
    221,
    37,
    169,
    170,
    30,
    192,
    155,
    232,
    36,
    110,
    132,
    181,
    74,
    239,
    62,
    222,
    215,
    171,
    73,
    40,
    93,
    246,
    81,
    15,
    35,
    150,
    123,
    95,
    78,
    194,
    254,
    0,
    44,
    18,
    150,
    189,
    105,
    1,
    0,
    0,
  ],
  "type": "Buffer",
}
`;

exports[`options asset 2`] = `null`;
